mm2src/coins/utxo/utxo_common.rs

```diff
@@ -3181,3 +3183,27 @@ fn test_tx_v_size() {
     let v_size = tx_size_in_v_bytes(&UtxoAddressFormat::Segwit, &tx);
     assert_eq!(v_size, 209)
 }
+
+pub fn get_raw_tx(
+    rpc_client: UtxoRpcClientEnum,
```

Comment:
Fns in this module typically accept a generic "coin" or `&UtxoCoinFields`, which should be sufficient here because all UTXO coins implement `AsRef<UtxoCoinFields>`.